### ISSUE TITLE
Unexpected EOF when using :range

### ISSUE THREAD

**Issue by atdixon** [kkinnear/zprint#190]
This is a rather obscure one, but I tried to pare it down to a small example:

`src/demo.clj`
```
(ns demo)

(def foo
  :foo
)

#_
  (def bar
    :bar
  )

(def cat :cat)
```

```
$ ./zprintm-1.1.1 '{:input {:range {:start 3 :end 7}}}' < src/demo.clj

Failed to zprint: java.lang.Exception: Unexpected EOF. [at line 6, column 11]
```

**Comment by kkinnear** [comment]
That's not good!  Thanks for submitting an issue for this.  Thanks as well for boiling it down, I can absolutely reproduce it and it is always nice to not have to wade through a lot extra stuff to focus on the real issue.  I'll let you know more when I've diagnosed it.

**Comment by kkinnear** [comment]
This is a fundamental problem with the parser I'm using for range handling.  I think I have a handle on a real fix, but that is going to come when I upgrade to a new overall parser, which is currently scheduled for the `1.1.3` release.  I'm presently  trying to wrap up the `1.1.2` release and get it out there, so I can start working on integrating the latest parser in `1.1.3`.

There is a possibility that I could find a kind of a hack of a workaround for this problem using the existing range parser, and get that into `1.1.2`.  If this is a critical issue for you, I'll look into that and see what I can come up with for `1.1.2` which is coming soon (or trying to).  If not, then I'll defer fixing this until I get to `1.1.3` and the better parser, which I expect will be the "real" fix for this issue.  Thanks again for bringing this up, I missed this entirely in my testing.

**Comment by atdixon** [comment]
Not critical/blocking for us. We can workaround. Thanks for the quick follow-up, as always!

**Comment by kkinnear** [comment]
Finally got this fixed by using the new parser.  Fixed in `1.2.0`!
